### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by CodingDoug**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,21 +3,50 @@
...
 npm install @libsql/client
 ```
 
-## Getting Started
+## Create a database client object
+
+### Importing
 
-Connecting to a local SQLite/libSQL database:
+There are multiple ways to import the module. For Node.js and other environments where want to use a local SQLite [file URL](#local-sqlite-file-urls), as well as network access to `sqld`:
```
Comment (honzasp) [inline README.md:21]: The "want" in the second sentence is missing a subject; perhaps this should be "where you want to use"?

Comment (honzasp) [reply README.md:21]: Also, Node.js is the *only* environment where local SQLite will work, there are no "other environments" distinct from Node that we can support.

File: README.md [hunk 2]
```diff
@@ -3,21 +3,50 @@
...
 
 ```typescript
-import { createClient } from "@libsql/client"
+import { createClient } from "@libsql/client";
+```
+
+For environments that don't have a local filesystem, but support HTTP or WebSockets, including:
```
Comment (honzasp) [inline README.md:27]: Strictly speaking, the `web` import should be used when the environment has *both* HTTP and WebSockets.

File: README.md [hunk 3]
```diff
@@ -26,34 +55,57 @@ const rs = await db.execute("SELECT * FROM users");
 console.log(rs);
 ```
 
-Connecting to a remote [libSQL server](https://github.com/libsql/sqld):
+### libSQL sqld instance
+
+To connect to a [libSQL sqld] instance using a [WebSocket URL](#websocket-urls) or [HTTP URL](#http-urls):
```
Comment (honzasp) [inline README.md:60]: This should perhaps mention that `libsql` implies TLS. This won't work with a sqld instance running locally (or, really, anything else but Turso).

**Comment by honzasp** [comment]
Hmm, this PR was merged before I had a chance to review :( Also, this repo uses bors, so it would be better to merge it using "bors merge" (not that it matters a lot).

File: README.md (hunk 2 above)
Comment (CodingDoug) [reply README.md:27]: Is that a hard requirement?  If I only have HTTP available, and I only use an HTTP URL, is there going to be a problem?  Or is the SDK free to use either one as it sees fit?

File: README.md (hunk 1 above)
Comment (CodingDoug) [reply README.md:21]: Will update in another PR.

File: README.md (hunk 3 above)
Comment (CodingDoug) [reply README.md:60]: The implication with the example below is that access to sqld might involve an auth token - it doesn't matter which transport is in use. I didn't really feel it was necessary to show a whole new example for HTTP when all that changes is the scheme of the URL.

I think what you're saying definitely belongs in the supported URLs section though, which I will definitely add.

File: README.md (hunk 2 above)
Comment (CodingDoug) [reply README.md:27]: A specific use case here would be the need to share a codebase that gets used in multiple environments, where the developer is dynamically deciding which URL to use. If in some cases they want a websocket and others in http, I'm expecting that this import would be OK.

Comment (honzasp) [reply README.md:27]: Yes, you are correct. However, if you only plan to use WebSocket, you can import just `@libsql/client/hrana`. Also, if your environment does not define `fetch()`, the `/http` and `/web` import may not work, but `/hrana` should work.

**Pull request merged**